tee_supplicant/src/tee_supplicant.c

```diff
@@ -377,6 +377,7 @@ static int tee_supp_init(const struct device *dev)
 	struct tee_version_info info = { 0 };
 	int rc;
 
+	LOG_ERR("supplicant initialized");
```

Comment:
Your commit message says "Updated copyright message" but you also making changes to the logging

Reply:
Sorry. I missed this